### PR TITLE
Database connection

### PR DESCRIPTION
**Pull request by kamilszczotka99**

### CODE REVIEW

File: Telstar Logistics Parcel Delivery Solution/Models/City.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+
+namespace Telstar_Logistics_Parcel_Delivery_Solution.Models
+{
+    [Keyless]
+    public class City
+    {
```
Comment (AndreasOksvold) [inline Telstar Logistics Parcel Delivery Solution/Models/City.cs:8]: Is this one missing the "disabled" field or did we scrap that?

File: Telstar Logistics Parcel Delivery Solution/Data/ApplicationDbContext.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+using Telstar_Logistics_Parcel_Delivery_Solution.Models;
+
+namespace Telstar_Logistics_Parcel_Delivery_Solution.Data
+{
+    public class ApplicationDbContext:DbContext
+    {
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options) { }
+        public DbSet<Account> Accounts { get; set; }
+        public DbSet<Category> Category { get; set; }
+        public DbSet<City> CITY { get; set; }
```
Comment (AndreasOksvold) [inline Telstar Logistics Parcel Delivery Solution/Data/ApplicationDbContext.cs:11]: Does it matter that "city" is all-caps?

**Review by AndreasOksvold — APPROVED** [review]
I'll approve it, but please take a look at my comments.

File: Telstar Logistics Parcel Delivery Solution/Data/ApplicationDbContext.cs (hunk 2 above)
Comment (kamilszczotka99) [reply Telstar Logistics Parcel Delivery Solution/Data/ApplicationDbContext.cs:11]: ive matched the names with the Telstar Database tables
image

File: Telstar Logistics Parcel Delivery Solution/Models/City.cs (hunk 1 above)
Comment (kamilszczotka99) [reply Telstar Logistics Parcel Delivery Solution/Models/City.cs:8]: I will add the Disabled field in another PR if thats fine :)

Comment (AndreasOksvold) [reply Telstar Logistics Parcel Delivery Solution/Models/City.cs:8]: That's great! 👍

File: Telstar Logistics Parcel Delivery Solution/Data/ApplicationDbContext.cs (hunk 2 above)
Comment (AndreasOksvold) [reply Telstar Logistics Parcel Delivery Solution/Data/ApplicationDbContext.cs:11]: Ok, then we have a system in place. Great! 👍

**Review by AndreasOksvold — APPROVED** [review]
Let's merge it!

**Review by AndreasOksvold — APPROVED** [review]
Looks like that didn't work. But now that I've been added as an approver it might!

**Review by AndreasOksvold — APPROVED** [review]

**Review by AndreasOksvold — APPROVED** [review]
Looks like there was a setting for the repository that was checked: Only Code Owners? Let's see now.

**Pull request merged**